### PR TITLE
Remove child

### LINKED ISSUE #218: Chapter/Skills child removal
## Motivations
Allow to remove a child from a chapter (see [designs](https://france-ioi.github.io/algorea-designs/03.Activities_Edit_05.Edit-chapter-content) and design branch)

## Subtasks
* Allow to select one or several item in the child list
* Add the "select all" button, make it work properly
* Add "Remove from this chapter" button.
* When clicking the button, remove the selected item from the list (do not call any service)

### PR DESCRIPTION
**Pull request by AyeRlS**
Fixes #218

Removing selected child on save.

### CODE REVIEW

File: src/app/modules/item/components/item-children-edit/item-children-edit.component.ts [hunk 1]
```diff
@@ -60,6 +61,11 @@ export class ItemChildrenEditComponent implements OnChanges {
     this.data.push({ title: child.title, type: child.type });
   }
 
+  deleteSelected(): void {
```
Comment (smadbe) [inline src/app/modules/item/components/item-children-edit/item-children-edit.component.ts:64]: Call it "remove", that's is more correct than "delete". And "*Selected" is not very clear either.
So call it, for instance, `onRemove`

File: src/app/modules/item/components/item-children-edit/item-children-edit.component.html [hunk 2]
```diff
@@ -3,16 +3,29 @@
...
               <td class="main">{{ rowData.title }}</td>
             </tr>
           </ng-template>
+          <ng-template pTemplate="summary">
+            <div class="summary-container">
+              <p-tableHeaderCheckbox></p-tableHeaderCheckbox>
+              <span class="select-all">Select all</span>
+              <div class="export-tools">
+                <span class="remove" (click)="deleteSelected()">
+                  <span>Remove</span>
```
Comment (smadbe) [inline src/app/modules/item/components/item-children-edit/item-children-edit.component.html:23]: Call it "Remove selected" and grey it out (and so disable it) when nothing is selected (as in pending request panel)

**Review by smadbe — CHANGES_REQUESTED** [review]
The selected row is not readable anymore :-/ We should try to fix that.
Can you do a proposal? Maybe just apply a light blue background instead?

File: src/app/modules/item/components/item-children-edit/item-children-edit.component.html (hunk 2 above)
Comment (AyeRlS) [reply src/app/modules/item/components/item-children-edit/item-children-edit.component.html]: Done

File: src/app/modules/item/components/item-children-edit/item-children-edit.component.ts (hunk 1 above)
Comment (AyeRlS) [reply src/app/modules/item/components/item-children-edit/item-children-edit.component.ts]: Done

**Comment by AyeRlS** [comment]
As discussed, I have set the background color when selected to match unselected items, and I haven't found a way to disable that behavior.

File: src/app/modules/item/components/item-children-edit/item-children-edit.component.html (hunk 2 above)
Comment (smadbe) [reply src/app/modules/item/components/item-children-edit/item-children-edit.component.html]: It is not greyed-out, it is not displayed at all. I would prefer greyed-out.

**Review by smadbe — CHANGES_REQUESTED** [review]
👍  except the greyed out

**Review by smadbe — APPROVED** [review]

**Pull request merged**